R/pkg/R/DataFrame.R

```diff
@@ -226,7 +226,8 @@ setMethod("showDF",
 
 #' show
 #'
-#' Print class and type information of a Spark object.
+#' If eager evaluation is enabled and the Spark object is a SparkDataFrame, return the data of
```

Comment:
return the data of the SparkDataFrame object -> evaluate the SparkDataFrame and print top rows of the SparkDataFrame

Reply:
Done